commit f044e2b482e083abafe7d31748c9db20fd0458f1
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Fri Oct 26 15:14:16 2018 +0100

    staging: vchiq_arm: Register bcm2835-codec as a platform driver
    
    Following the same pattern as bcm2835-camera and bcm2835-audio,
    register the V4L2 codec driver as a platform driver
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
index d3b5a00d2c53..7b07d6e147f1 100644
--- a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
+++ b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
@@ -171,6 +171,7 @@ static struct device *vchiq_dev;
 static DEFINE_SPINLOCK(msg_queue_spinlock);
 static struct platform_device *bcm2835_camera;
 static struct platform_device *bcm2835_audio;
+static struct platform_device *bcm2835_codec;
 
 static const char *const ioctl_names[] = {
 	"CONNECT",
@@ -3660,6 +3661,9 @@ static int vchiq_probe(struct platform_device *pdev)
 	bcm2835_audio = vchiq_register_child(pdev, "bcm2835_audio");
 	if (IS_ERR(bcm2835_audio))
 		bcm2835_audio = NULL;
+	bcm2835_codec = vchiq_register_child(pdev, "bcm2835-codec");
+	if (IS_ERR(bcm2835_codec))
+		bcm2835_codec = NULL;
 
 	return 0;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1383_f044e2b482e083abafe7d31748c9db20fd0458f1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1383_f044e2b482e083abafe7d31748c9db20fd0458f1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
